__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require("supertest")
+const app = require("../app.js")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed.js")
+const data = require("../db/data/test-data")
+
+beforeEach(() => {
+    return seed(data);
+  });
+
+  afterAll(() => {
+    db.end();
+  });
+
+describe("/api/topics", () => {
+    test("GET 200: Should respond with a 200 OK status", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+    })
+    test("GET 200: Should respond with all topics", () => {
+        return request(app)
+        .get("/api/topics")
+        .then(({body}) => {
+            expect(body).toEqual(    
```

Comment:
this test may be quite brittle if the test data ever changes - try to loop through the array received and check that each object has what it should - look into the jest matchers .toMatchObject to help with this